src/Console/SessionsCommand.php

```diff
@@ -39,26 +46,67 @@ public function handle(DatabaseManager $db): void
             static fn(string $name): bool => config("database.connections.{$name}.driver") === 'spanner',
         );
 
-        $headers = ['Name', 'CreatedAt', 'LastUsedAt'];
-
         foreach ($spannerConnectionNames as $name) {
             $connection = $db->connection($name);
             if ($connection instanceof SpannerConnection) {
-                $sessions = $connection->listSessions();
-                $count = count($sessions);
-                $data = [];
-                $this->info("{$connection->getName()} contains {$count} session(s).");
-                foreach ($sessions as $session) {
-                    $data[] = [
-                        $session->getName(),
-                        $session->getCreatedAt(),
-                        $session->getLastUsedAt(),
-                    ];
-                }
-                if (count($data) > 0) {
-                    $this->table($headers, $data);
+                $sessions = $this->makeSessionData($connection);
+                $this->info("{$connection->getName()} contains {$sessions->count()} session(s).");
+                if ($sessions->isNotEmpty()) {
+                    $headers = array_keys($sessions[0]);
+                    $this->table($headers, $sessions);
                 }
             }
         }
     }
+
+    /**
+     * @param Connection $connection
+     * @return Collection<int, array{ Name: string, Created: string, LastUsed: string }>
+     */
+    protected function makeSessionData(Connection $connection): Collection
+    {
+        $descending = $this->getOrder() === 'desc';
+
+        return $connection->listSessions()
+            ->sortBy(fn(Session $s) => $this->getSortValue($s), descending: $descending)
+            ->map(static fn(Session $s) => [
+                'Name' => $s->getName(),
+                'Created' => $s->getCreatedAt()->format('Y-m-d H:i:s'),
+                'LastUsed' => $s->getLastUsedAt()->format('Y-m-d H:i:s'),
+            ]);
+    }
+
+    /**
+     * @param Session $session
+     * @return string
+     */
+    protected function getSortValue(Session $session): string
+    {
+        $sort = $this->option('sort');
+        assert(is_string($sort));
+        return match (Str::camel($sort)) {
```

Comment:
I think that this should be not lower camel case but upper camel case.

```suggestion
        return match (Str:: studly($sort)) {
```